catalog-info.yaml

```diff
@@ -18,6 +18,6 @@ spec:
   lifecycle: production
   consumesApis:
     - api:hmpps-interventions
-    # - api:hmpps-assess-risks-and-needs # not yet defined
+    - api:hmpps-risks-and-needs
+    - api:hmpps-community
     # - api:hmpps-auth # not yet defined
```

Comment:
`api:hmpps-auth` is proposed in https://github.com/ministryofjustice/hmpps-auth/pull/1220